### ISSUE TITLE
Group chat > It is able to create a group chat with more than 10 members

### ISSUE THREAD

**Issue by tlle14** [linagora/Twake-Mobile#1035]
# Precondition
The company has more than 10 members

# Actual Result
On Mobile, it is able to create a group chat with more than 10 members
https://user-images.githubusercontent.com/69950655/138807212-50556b14-88f6-496f-bd63-53cf8bed5d38.mp4

# Expected Result
It should allow creating a group with maximum 10 members. Else, suggest the user create a channel instead.
Please follow the business rule on Web version
![image](https://user-images.githubusercontent.com/69950655/138806828-09578dd5-31db-4e73-af14-d694e513c6c9.png)

# Environment
Build version: 1.0.1
Device:
Samsung Galaxy J7 - Android 8.1.0 - 5.5 inch
iPhone 11 Promax - iOS 14.4 - 6.46 inch
QA environment

**Comment by hoangdat** [comment]
@BenoitTallandier can you mention us in the story or requirement for `create chat` with limitation members?

**Comment by BenoitTallandier** [comment]
Yes, we can invite 10 people maximum in a `group chat`, either on web or mobile app.

**Comment by hoangdat** [comment]
> Yes, we can invite 10 people maximum in a `group chat`, either on web or mobile app. There is no US on mobile for that, do you need that I create one ? :)

yes, please help me create one. Moreover as I checked in web, user can create a group in case he wants to add more than 10 people. Hope we will have UI design for that also.

But IMO, this issue is not `Major`, we can prioritize other issues.

**Comment by EvGen94** [comment]
@BastiaanVanGaalen Hi! Please create a snackbar or something like that to show the limit - 10 users for the direct chat

**Comment by BenoitTallandier** [comment]
To be precise : I can't invite more than 10 users inside a direct discussion. So the total is 11 (10 members and me).

**Comment by BastiaanVanGalen** [comment]
@EvGen94  https://zpl.io/g8qnLKA
